driverstation.py

```diff
@@ -53,6 +58,20 @@ def joystick_stats(joystick):
 
     Look at the documentation for NetworkTables for some ideas.
          https://robotpy.readthedocs.io/projects/pynetworktables/en/latest/examples.html
+    """
+
+    for i in range(len(buttons)):
+        buttons[i] = joystick.get_button(i)
+        
+
+    for j in range(len(axis_values)):
+        axis_values[j] = joystick.get_axis(j)
+    
+    xbc_nt.putNumberArray("Buttons", buttons)
+    xbc_nt.putNumberArray("Axis", axis_values)
+
+
+
     """
```

Comment:
Yeah, I think we can get rid of this event based stuff and just plan to push the full state of the controller once every robot cycle.